CHANGELOG.md

```diff
@@ -64,6 +64,14 @@ This was added in [pull request #2230: Add extra letter spacing modifier for inp
 
 ### Deprecated features
 
+#### Stop using JavaScript API properties other than the `init` method
+
+All JavaScript properties in the API for our JavaScript components except for the `init` method are now deprecated, and will be made private in v5.0.
```

Comment:
```suggestion
We have deprecated all JavaScript properties in the API for our JavaScript components, except for the `init` method. We will make all deprecated JavaScript properties private in v5.0.
```

Reply:
What about this, to avoid some of the repetition? (Especially 'JavaScript' which was already in there twice…)

> We have deprecated all of the JavaScript properties in the API except for the `init` method. We will make them private in v5.0.

Reply:
I repeated as the last thing mentioned in the previous sentence in the init method, which means grammatically 'them' would refer to the init method. I shall shuffle things...

Reply:
I've amended my suggestion - does that work?

Reply:
I think the version with the repetition was better – this one seems to slightly contradict itself. What about:

> We have deprecated all of the JavaScript properties in the API, except for the `init` method for each component. We will make all the deprecated JavaScript properties private in v5.0.

Reply:
Trying to break it down:

- We have deprecated all of the JavaScript properties in the API.
- The exception to this is the `init` method for each component, which remains public.

With the additional comma I think it would read as:

- We have deprecated all of the JavaScript properties in the API for every component.
- The exception to this is the `init` method, which remains public.

Which isn't quite right – every component has an `init` method, which will remain public; and there are also parts of the API that aren't part of any component.

Reply:
So I've changed the suggestion to your 3 sentences.